### DISCUSSION TITLE
Collateralization & punishment

### DISCUSSION THREAD

**Discussion by xPi2** [nation3/court#11] [General]
At least from my point of view, the success of this system depends on a very basic primitive: safe agreements between two or more non-trusted parties.

How do we achieve this?
A. Disputes over an agreement can be arbitrated by a trusted 3rd party (the Court).
B. Non-compliance with what was agreed or other types of dangerous behavior can be punished.

I want to focus on the second half.

The most common mechanism to prevent people from playing by the rules is collateralization.
A straightforward example in crypto is lending markets. You borrow money from other/s users by agreeing to return it in the future. To do so, you deposit collateral bigger than the amount you are borrowing that will cover your debt if you breach the agreement. For example, by refusing to repay your debt, don't keep your position under specific parameters or skip an interest payment.

Most agreements work the same way in real life, but with a big difference: you don't have to lock your collateral in advance. If you breach an agreement, the authority of the agreement's jurisdiction, usually the court system of nation-states, will use their power to punish you.
This system sometimes has disadvantages but allows us to participate in more agreements than we think beforehand. The bank doesn't have to trust you because it trusts the judicial system. If you don't repay your debt, the state will give them the right to take away your properties until your debt is paid.

Building escrows in which you deposit agreed collateral that can be taken away from you if you don't comply with the terms of the agreement is straightforward. Allowing to do this while keep being able to use that collateral is the tricky part but the most exciting part.

---

When you participate in the economy of a nation-state, you agree to play by a set of rules that keep the system working. Those rules usually include giving the authority of the system the right to punish you in some way in exchange.

Could we achieve some fair, more straightforward equivalent arrangement for Nation3?
I think we have already implemented something close by now. When you deposit your capital through NATION on the voting escrow contract, you are giving up the ability to freely use that capital in exchange for the power to steer the nation's decisions.
We keep trying to increase the value you can get from that capital:
- You can boost your rewards for providing liquidity for the NATION
- You have preferential access to paid national work, either through Dework tasks or guilds budget allocation.
- You can claim a membership token that hopefully will allow you to access community events and other spaces soon.

What about adding a new dimension of value to that capital? Use it as valid collateral to use for agreements on Nation3 jurisdiction.
Can we achieve this with the current escrow implementation? I don't think so. After all, its name is self-explanatory on its purpose, "voting escrow"; good for aligning voting power to long commitment, not so good for composability. The most significant limitation is that other contracts can't manage the lock, even allow-listed ones. This means no transfers and no arbitrated adjustments (punishments) are possible.

How can we achieve a new kind of "governed escrow" that can be used for more than voting while keeping the alignment incentives & the security of the voting escrow?

Let the discussion begins.

**Comment by gregfromstl** [comment]
I'm struggling to think of any way we can provide this sort of utility while also keeping the funds secure. The heart of any currency's value is its transferability, but we can't allow any sort of transferring otherwise we risk the funds not being available when the party breaches the agreement.

One option to at least soften the blow of locking up NATION during an agreement could be to auto-enter any locked NATION into the liquidity pool, and give the rewards to those in the agreement. This at least "puts their money to work" rather than having to sit on the sidelines as long as they're in the agreement.
